supervisor/docker/interface.py

```diff
@@ -19,6 +25,8 @@
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
 IMAGE_WITH_HOST = re.compile(r"^((?:[a-z0-9]+(?:-[a-z0-9]+)*\.)+[a-z]{2,})\/.+")
+DOCKER_HUB = "hub.docker.com"
+DOCKER_HUB_REGISTRY = "https://index.docker.io/v1/"
```

Comment:
Is that not v2?

Reply:
CLI says v1, the docker python package we uses says v1.

But I'll update to actually not use it